tests/manager/manager-executeBulkWrite-002.phpt

```diff
@@ -1,5 +1,5 @@
 --TEST--
-MongoDB\Driver\Manager::executeBulkWrite() with duplicate key errors (ordered)
+MongoDB\Driver\Manager::executeBulkWrite() with upserted ids
```

Comment:
The "duplicate key errors (ordered)" and "duplicate key errors (unordered)" tests were renamed to `manager-executeBulkWrite_error-001.phpt` and `manager-executeBulkWrite_error-002.phpt`, respectively. Git didn't pick up that it was a rename, though, as I ported over the single write method tests to `executeBulkWrite()` and kept the filename numbers in sequence.